functions/APIs/deals.js

```diff
@@ -1,11 +1,14 @@
+const express = require("express");
+const dealsRoute = express.Router();
+const auth = require("../util/auth");
 const { db } = require("../util/admin");
 
 /*
 read a list of all deals
 GET /deals
 success response: array of deal objects (documented in POST /deals)
 */
```

Comment:
```suggestion
/**
 * Read a list of all deals
 * GET /deals
 * Success response: array of deal objects (documented in POST /deals)
 */
```

https://google.github.io/styleguide/jsguide.html#jsdoc-general-form

Since this is on already checked in code, feel free to address (everywhere, not just here) in another PR to keep this one focused.